easybuild/easyconfigs/g/Gaussian/Gaussian-09.e.01-AVX.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'Tarball'
+
+# IMPORTANT!!
+# Needs to be installed with --umask=007
+
+name = 'Gaussian'
+version = '09.e.01'
+versionsuffix = '-AVX'
+
+homepage = 'https://www.gaussian.com/'
+
+# For the sake of users I advise changing the last line in the description
+# and versionsuffix to match the actual installed version.
+description = """
+Gaussian provides state-of-the-art capabilities for electronic structure
+modeling. Gaussian 09 is licensed for a wide variety of computer
+systems. All versions of Gaussian 09 contain every scientific/modeling
+feature, and none imposes any artificial limitations on calculations
+other than your computing resources and patience.
+
+This is the official gaussian AVX build.
+"""
+
+toolchain = SYSTEM
+
+# Gaussian is licensed software.
+# You need to pick the tar file of your choice from the DVD,
+# put it in your sources tree (should be a tree that ordinary users can't read)
+# and change the name in "sources"
+sources = ['09.e.01-AVX-A64-114X.tgz']
+checksums = ['af1e1cdf7d2799690a642b27ee0779b7cca9695b9a86a48739f3d9fc8c54a98d']
+
+start_dir = '..'
+
+postinstallcmds = [
+    'mkdir %(installdir)s/bin',
+    'ln -s ../g%(version_major)s/g%(version_major)s ../g%(version_major)s/ghelp %(installdir)s/bin',
+    'chmod -R o-rwx %(installdir)s/g%(version_major)s',
+]
+
+sanity_check_paths = {
+    'files': ['g%(version_major)s/g%(version_major)s', 'g%(version_major)s/l9999.exe'],
+    'dirs': ['g%%(version_major)s/%s' % x for x in ['bsd', 'tests', 'basis', 'doc']],
+}
+
+modextravars = {
+    'g%(version_major)sroot': '%(installdir)s',
+    'G%(version_major)sBASIS': '%(installdir)s/g%(version_major)s/basis',
+    'GAUSS_BSDDIR': '%(installdir)s/g%(version_major)s/bsd',
+    '_DSM_BARRIER': 'SHM',
+    'PGI_TERM': 'trace,abort',
+}
+
+modextrapaths = {
+    'GAUSS_EXEDIR': ['g%(version_major)s', 'g%(version_major)s/bsd'],
+    'PATH': ['g%(version_major)s/bsd', 'g%(version_major)s'],
+    'LD_LIBRARY_PATH': 'g%(version_major)s',
+}
+
+modaliases = {'sl': '%(installdir)s/g%(version_major)s/tests/searchlog.csh'}
```

Comment:
This looks like a site customization to me, doesn't belong here?

Reply:
Taken from #7553, it's usable for users who want to search the output with that script.

Reply:
But it's way too generic imho...

Should be `gaussian_searchlog` or `g09_searchlog` or `g09_sl`

Reply:
Yeah, much better. But since I've never heard a user ask for this at our site I think I'll drop it.

Reply:
Makes sense to me.

Reply:
Left it in but commented out and with a short explanation.